README.md

```diff
@@ -78,7 +80,7 @@ experiment.log_metric("accuracy", accuracy)
 Then explore the project by running the dashboard:
 
 ```
-rubicon ui --root-dir /rubicon-root
+rubicon_ml ui --root-dir /rubicon-root
```

Comment:
I didn't change the CLI command - should I?

Reply:
probably should have the cli command match the package name -- i could change that in this PR